onnx_xla/onnx_xla_interface.cc

```diff
@@ -267,3 +277,79 @@ ONNXIFI_SYMBOL_NAME(onnxReleaseGraph)(onnxGraph graph) {
   }
   ONNXIFI_CATCH_EXCPETION();
 }
+
+//Initialize event by creating EventControl object on the heap
+ONNXIFI_PUBLIC ONNXIFI_CHECK_RESULT onnxStatus ONNXIFI_ABI ONNXIFI_SYMBOL_NAME(
+    onnxInitEvent)(onnxBackend backend, onnxEvent* event) {
+  try {
+    if (!backend)  {
+      return ONNXIFI_STATUS_INVALID_BACKEND;
+    }
+    if (!event)  {
+      return ONNXIFI_STATUS_INVALID_POINTER;
+    }
+
+    *event = (onnxEvent) new EventControl();
```

Comment:
Please use C++-style cast, i.e.

`*event = static_cast<onnxEvent>(new EventControl());`

Reply:
`reinterpret_cast` would be fine too (`static_cast` works because `onnxEvent` is `void*`, but thats a hack)

Reply:
changed to reinterpret_cast, since reinterpret_cast used elsewhere in the file